packages/i3ipc/i3ipc.0.3/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "armael@isomorphis.me"
+authors: "Armaël Guéneau"
+homepage: "https://github.com/Armael/ocaml-i3ipc"
+bug-reports: "https://github.com/Armael/ocaml-i3ipc/issues"
+license: "MIT"
+tags: ["i3" "ipc" "window-manager"]
+dev-repo: "git+https://github.com/Armael/ocaml-i3ipc"
+doc: "https://armael.github.io/ocaml-i3ipc/0.3/"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "lwt"
+  "lwt_ppx"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.8"}
```
The `dune-project` file in your sources says this package requires at least dune 1.8

Reply:
Good catch. I've instead relaxed the constraint on the version in the dune-project; it now should work for dune >= 1.0, ie any version of dune.